artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/Page.java

```diff
@@ -253,8 +253,8 @@ public synchronized PagedMessage readMessage(int startOffset,
 
    public synchronized List<PagedMessage> read(StorageManager storage, boolean onlyLargeMessages) throws Exception {
       if (logger.isDebugEnabled()) {
-         logger.debugf("reading page %d on address = %s onlyLargeMessages = %b", storeName, pageId,
-                       storage, onlyLargeMessages);
+         logger.debugf("reading page %d on address = %s onlyLargeMessages = %b", new Object[] {pageId,
```

Comment:
why the Object array?

Reply:
I had the same question. :smile:

Reply:
that's like changing 6 for half dozen...
It's the same thing.. I don't understand what is the point?

Reply:
Otherwise I get a conflict because there are two methods with similar signatures:
`debugf(String, int, Object, Object)`
`debugf(String, Object, Object, Object)`

Reply:
I see thanks